template/Makefile.in

```
@@ -493,9 +493,8 @@ clean-local::
 	  enc/encinit.c enc/encinit.$(OBJEXT) $(pkgconfig_DATA) \
 	  ruby-runner.$(OBJEXT) ruby-runner.h \
 	|| $(NULLCMD)
-	@$(RM) $(ALLOBJS:.$(OBJEXT)=.bc)
-	@$(RM) $(ALLOBJS:.$(OBJEXT)=.i)
-	@$(RM) $(ALLOBJS:.$(OBJEXT)=.s)
+	$(Q)find . -depth \( -name '*.bc' -o -name '*.[is]' \) -delete 2> /dev/null || true
+	-$(Q)$(RMALL) *.dSYM
```

Unfortunately, `-delete` is not standardized in POSIX find.

```suggestion
	$(Q)find . ! -type d \( -name '*.bc' -o -name '*.[is]' \) -exec rm -f {} + || true
```

Since we don't expect to delete directories here, `-depth` is not needed.

Does `find` print any error or warning messages?

---

I made this change and tested it on macos and linux. I didn't see any errors or warnings. It also deleted the correct files. 👍🏼